### ISSUE TITLE
The documentation is incorrect

### ISSUE THREAD

**Issue by sukhwinder33445** [Icinga/icinga2#9383]
## Describe the bug

The installation of icinga2 according to the documentation initializes two databases (`icinga`, `icinga2`) which have the same tables.

Link:
https://icinga.com/docs/icinga-2/latest/doc/02-installation/01-Debian/#install-ido-feature

* command `apt-get install icinga2-ido-mysql` initialize the `icinga2` database.
* [Next step](https://icinga.com/docs/icinga-2/latest/doc/02-installation/01-Debian/#set-up-mysql-database) initializes the `icinga` database with the same tables.

**Comment by Al2Klimov** [comment]
Colleagues, shall we install dbconfig-no-thanks along with icinga2-ido-$RDBMS?

**Comment by julianbrost** [comment]
No, having dbconfig stuff in the package and then installing another package to disable it (which affects other packages apart from Icinga as well) makes no sense.

Why not just label the section as optional in the sense that you only have to do this if you require full control over the database setup and have opted to not automatically configure the database during the installation of the package?
